lua/neoclip/telescope.lua

```diff
@@ -147,6 +147,9 @@ local function get_export(register_names, typ)
                 end,
                 dyn_title = function(self, entry)
                     return spec_from_entry(entry)
+                end,
+                get_buffer_by_name = function(self, entry)
+                    return 'test'
```

Comment:
I think this should be unique by entry.

`return "neoclip_" .. entry.idx` I think works (I looked briefly into this a couple of days ago).

This somewhat speeds up the preview for my; however it's still rather slow for some reason. I'll try to think about this some more.